internal/pkg/cli/env_upgrade.go

```diff
@@ -284,7 +284,8 @@ func (o *envUpgradeOpts) upgradeEnvironment(upgrader envUpgrader, conf *config.E
 	if err := upgrader.UpgradeEnvironment(&deploy.CreateEnvironmentInput{
 		Version: toVersion,
 		App: deploy.AppInformation{
-			Name: conf.App,
+			Name:   conf.App,
+			Domain: app.Domain,
```

Comment:
should we check that `conf.App == app.Name`?

Reply:
I'll just use `app.Name`